### ISSUE TITLE
Cant call `users calendar-view list`

### ISSUE THREAD

**Issue by danriedl** [microsoftgraph/msgraph-cli#389]
I do want to check my next meetings in the calendar with the msgraph-cli to be notified in my Zellij session.
But somehow i cant do following command:
![image](https://github.com/microsoftgraph/msgraph-cli/assets/107212962/4273b481-1d2f-4d61-a38c-401b9c4fd46a)

```bash
./mgc users calendar-view list --user-id xxxyyyzz-xxyy-zzzz-xxyy-b01b21dea883 --start-date-time "2023-11-10T09:58:07.307Z" --end-date-time "2023-11-10T19:58:07.307Z" | jq
{
  "error": {
    "code": "ErrorInvalidParameter",
    "message": "This request requires a time window specified by the query string parameters StartDateTime and EndDateTime."
  }
}
```

**Comment by calebkiage** [comment]
Hello @danriedl, what is the output when you add the `--debug` option?

**Comment by danriedl** [comment]
```text
[Informational] Azure-Identity: DeviceCodeCredential.GetToken invoked. Scopes: [ https://graph.microsoft.com/.default ] ParentRequestId:
[Informational] Azure-Identity: False MSAL 4.56.0.0 MSAL.NetCore .NET 7.0.12 Darwin 22.6.0 Darwin Kernel Version 22.6.0: Wed Oct  4 21:25:26 PDT 2023; root:xnu-8796.141.3.701.17~4/RELEASE_X86_64 [2023-11-16 07:05:55Z - xxxxxxxx-yyyy-zzzz-xxxx-c585f04a8425] MSAL MSAL.NetCore with assembly version '4.56.0.0'. CorrelationId(xxxxxxxx-yyyy-zzzz-xxxx-c585f04a8425)
[Informational] Azure-Identity: False MSAL 4.56.0.0 MSAL.NetCore .NET 7.0.12 Darwin 22.6.0 Darwin Kernel Version 22.6.0: Wed Oct  4 21:25:26 PDT 2023; root:xnu-8796.141.3.701.17~4/RELEASE_X86_64 [2023-11-16 07:05:55Z - xxxxxxxx-yyyy-zzzz-xxxx-c585f04a8425] === AcquireTokenSilent Parameters ===
[Informational] Azure-Identity: False MSAL 4.56.0.0 MSAL.NetCore .NET 7.0.12 Darwin 22.6.0 Darwin Kernel Version 22.6.0: Wed Oct  4 21:25:26 PDT 2023; root:xnu-8796.141.3.701.17~4/RELEASE_X86_64 [2023-11-16 07:05:55Z - xxxxxxxx-yyyy-zzzz-xxxx-c585f04a8425] LoginHint provided: False
[Informational] Azure-Identity: False MSAL 4.56.0.0 MSAL.NetCore .NET 7.0.12 Darwin 22.6.0 Darwin Kernel Version 22.6.0: Wed Oct  4 21:25:26 PDT 2023; root:xnu-8796.141.3.701.17~4/RELEASE_X86_64 [2023-11-16 07:05:55Z - xxxxxxxx-yyyy-zzzz-xxxx-c585f04a8425] Account provided: True
[Informational] Azure-Identity: False MSAL 4.56.0.0 MSAL.NetCore .NET 7.0.12 Darwin 22.6.0 Darwin Kernel Version 22.6.0: Wed Oct  4 21:25:26 PDT 2023; root:xnu-8796.141.3.701.17~4/RELEASE_X86_64 [2023-11-16 07:05:55Z - xxxxxxxx-yyyy-zzzz-xxxx-c585f04a8425] ForceRefresh: False
[Informational] Azure-Identity: False MSAL 4.56.0.0 MSAL.NetCore .NET 7.0.12 Darwin 22.6.0 Darwin Kernel Version 22.6.0: Wed Oct  4 21:25:26 PDT 2023; root:xnu-8796.141.3.701.17~4/RELEASE_X86_64 [2023-11-16 07:05:55Z - xxxxxxxx-yyyy-zzzz-xxxx-c585f04a8425]
=== Request Data ===
Authority Provided? - True
Scopes - https://graph.microsoft.com/.default
Extra Query Params Keys (space separated) -
ApiId - AcquireTokenSilent
IsConfidentialClient - False
SendX5C - False
LoginHint ? False
IsBrokerConfigured - False
HomeAccountId - False
CorrelationId - xxxxxxxx-yyyy-zzzz-xxxx-c585f04a8425
UserAssertion set: False
LongRunningOboCacheKey set: False
Region configured:

[Informational] Azure-Identity: False MSAL 4.56.0.0 MSAL.NetCore .NET 7.0.12 Darwin 22.6.0 Darwin Kernel Version 22.6.0: Wed Oct  4 21:25:26 PDT 2023; root:xnu-8796.141.3.701.17~4/RELEASE_X86_64 [2023-11-16 07:05:55Z - xxxxxxxx-yyyy-zzzz-xxxx-c585f04a8425] === Token Acquisition (SilentRequest) started:
	 Scopes: https://graph.microsoft.com/.default
	Authority Host: login.microsoftonline.com
[Informational] Azure-Identity: False MSAL 4.56.0.0 MSAL.NetCore .NET 7.0.12 Darwin 22.6.0 Darwin Kernel Version 22.6.0: Wed Oct  4 21:25:26 PDT 2023; root:xnu-8796.141.3.701.17~4/RELEASE_X86_64 [2023-11-16 07:05:55Z - xxxxxxxx-yyyy-zzzz-xxxx-c585f04a8425] [Region discovery] Not using a regional authority.
[Informational] Azure-Identity: False MSAL 4.56.0.0 MSAL.NetCore .NET 7.0.12 Darwin 22.6.0 Darwin Kernel Version 22.6.0: Wed Oct  4 21:25:26 PDT 2023; root:xnu-8796.141.3.701.17~4/RELEASE_X86_64 [2023-11-16 07:05:55Z - xxxxxxxx-yyyy-zzzz-xxxx-c585f04a8425] Access token is not expired. Returning the found cache entry. [Current time (11/16/2023 07:05:55) - Expiration Time (11/16/2023 08:07:38 +00:00) - Extended Expiration Time (11/16/2023 08:07:38 +00:00)]
[Informational] Azure-Identity: False MSAL 4.56.0.0 MSAL.NetCore .NET 7.0.12 Darwin 22.6.0 Darwin Kernel Version 22.6.0: Wed Oct  4 21:25:26 PDT 2023; root:xnu-8796.141.3.701.17~4/RELEASE_X86_64 [2023-11-16 07:05:55Z - xxxxxxxx-yyyy-zzzz-xxxx-c585f04a8425] Returning access token found in cache. RefreshOn exists ? False
[Informational] Azure-Identity: False MSAL 4.56.0.0 MSAL.NetCore .NET 7.0.12 Darwin 22.6.0 Darwin Kernel Version 22.6.0: Wed Oct  4 21:25:26 PDT 2023; root:xnu-8796.141.3.701.17~4/RELEASE_X86_64 [2023-11-16 07:05:55Z - xxxxxxxx-yyyy-zzzz-xxxx-c585f04a8425] [Region discovery] Not using a regional authority.
[Informational] Azure-Identity: False MSAL 4.56.0.0 MSAL.NetCore .NET 7.0.12 Darwin 22.6.0 Darwin Kernel Version 22.6.0: Wed Oct  4 21:25:26 PDT 2023; root:xnu-8796.141.3.701.17~4/RELEASE_X86_64 [2023-11-16 07:05:55Z - xxxxxxxx-yyyy-zzzz-xxxx-c585f04a8425]
	=== Token Acquisition finished successfully:
[Informational] Azure-Identity: False MSAL 4.56.0.0 MSAL.NetCore .NET 7.0.12 Darwin 22.6.0 Darwin Kernel Version 22.6.0: Wed Oct  4 21:25:26 PDT 2023; root:xnu-8796.141.3.701.17~4/RELEASE_X86_64 [2023-11-16 07:05:55Z - xxxxxxxx-yyyy-zzzz-xxxx-c585f04a8425]  AT expiration time: 16.11.2023 08:07:38 +00:00, scopes: openid profile email https://graph.microsoft.com/User.Read https://graph.microsoft.com/User.Read.All https://graph.microsoft.com/.default. source: Cache
[Informational] Azure-Identity: DeviceCodeCredential.GetToken succeeded. Scopes: [ https://graph.microsoft.com/.default ] ParentRequestId:  ExpiresOn: 2023-11-16T08:07:38.0000000+00:00
dbug: Microsoft.Graph.Cli.Core.Http.LoggingHandler[1]

      Request:

      GET https://graph.microsoft.com/v1.0/users/xxxxxxxx-yyyy-zzzz-xxxx-yyyyyydea883/calendarView HTTP/1.1
      Accept: application/json; q=1
      Authorization: [PROTECTED]
      FeatureFlag: 00000043
      Cache-Control: no-store, no-cache
      User-Agent: kiota-dotnet/1.1.1
      SdkVersion: graph-cli-1.0/1.0.0, graph-dotnet-core/3.0.11 (featureUsage=0000002B; hostOS=Unix 13.6.1; hostArch=X64; runtimeEnvironment=.NET 7.0.12;)
      client-request-id: xxxxxxxx-yyyy-zzzz-xxxx-42e2590f7bf2



dbug: Microsoft.Graph.Cli.Core.Http.LoggingHandler[2]

      Response:

      HTTP/1.1 400 Bad Request
      Cache-Control: private
      Transfer-Encoding: chunked
      Vary: Accept-Encoding
      Strict-Transport-Security: max-age=31536000
      request-id: xxxxxxxx-yyyy-zzzz-xxxx-7376490078fc
      client-request-id: xxxxxxxx-yyyy-zzzz-xxxx-42e2590f7bf2
      x-ms-ags-diagnostic: {"ServerInfo":{"DataCenter":"Germany West Central","Slice":"E","Ring":"5","ScaleUnit":"000","RoleInstance":"FR1PEPF000007AC"}}
      Date: Thu, 16 Nov 2023 07:05:54 GMT
      Content-Type: application/json; odata.metadata=minimal; odata.streaming=true; IEEE754Compatible=false; charset=utf-8
      Content-Encoding: gzip

      {"error":{"code":"ErrorInvalidParameter","message":"This request requires a time window specified by the query string parameters StartDateTime and EndDateTime."}}

{
  "error": {
    "code": "ErrorInvalidParameter",
    "message": "This request requires a time window specified by the query string parameters StartDateTime and EndDateTime."
  }
}

```

**Comment by calebkiage** [comment]
This issue should be fixed in [v1.1.0](https://github.com/microsoftgraph/msgraph-cli/releases/tag/v1.1.0)
